airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/accounts.json

```diff
@@ -80,7 +80,7 @@
       "type": ["null", "number"]
     },
     "PauseReason": {
-      "type": ["null", "string"]
+      "type": ["null", "number"]
```

Comment:
Oh Microsoft - looks like this is a number (unsignedByte) that can be used to look up a reason later - https://docs.microsoft.com/en-us/advertising/customer-management-service/advertiseraccount?view=bingads-13

Good catch!